docs/user-guide/server-side-install-overview.md

```diff
@@ -1,19 +1,124 @@
+
+
+For more information about the Zowe cross memory server, see [Installing and configuring the Zowe cross memory server (ZWESISTC)](./user-guide/configure-xmem-server).
+
+## Zowe runtime installation methods and roadmap
+
+To get started with installing Zowe runtime, select your installation method from the following options. (Details of each installation method are described later in this article).
+
+* SMP/E build
+* Convenience build
+* Portable software instance (PSWI)
+* Containerization build  
+    **Note:** The containerization build makes it possible for some UNIX components to run in a container rather than under USS. For more information about this build method, see [Installing Zowe Containers](k8s-introduction.md).
+
+Zowe server-side components can be installed either on z/OS only, or both on z/OS and on Docker.
+
+The following diagram presents a roadmap for installation using the three primary installation methods. If you plan to install these components using a container build, see Installing Zowe Containers.
+
+![Install the Zowe z/OS runtime](../images/install/install.png)
+
+The Zowe z/OS binaries are distributed in the following installation formats. Each of these  formats contain the same contents, but the installation methods vary. 
+### Details of runtime installation methods
+
+Select from the following methods to install the Zowe server-side components on z/OS. 
+<!-- Add the advantages and disadvantanges to each method -->
+
+   - **SMP/E build**
+
+     The Zowe z/OS binaries are packaged as the following files that you can download. You install this build through SMP/E.  
+     - A pax.Z file, which contains an archive (compressed copy) of the FMIDs to be installed.
+     - A readme file, which contains a sample job to decompress the pax.Z file, transform it into a format that SMP/E can process, and invoke SMP/E to extract and expand the compressed SMP/E input data sets.
+
+     To obtain and install the SMP/E build, see [Installing Zowe SMP/E](install-zowe-smpe.md).
+
+   - **Convenience build**
+
+     Zowe z/OS binaries are packaged as a PAX file which is a full product install. Transfer this PAX file to a USS directory and expand its contents. Use the [zwe](installandconfig.md#zwe-server-command) command `zwe install` to extract a number of PDS members which contain load modules, JCL scripts, and PARMLIB entries. 
+
+     To obtain and install the convenience build, see [Installing Zowe runtime from a convenience build](install-zowe-zos-convenience-build.md).
+
+   - **Portable Software Instance (PSWI)**
```

Comment:
This is preferred method, shouldn't it be the first one?

Reply:
This is the order that is presneted in the diagram.